src/geo/map/cartodb-layer.js

```diff
@@ -5,7 +6,7 @@ var MapLayer = require('./map-layer');
 var CartoDBLayer = MapLayer.extend({
 
   defaults: {
-    attribution: config.get('cartodb_attributions'),
+    attribution: cdb.config.get('cartodb_attributions'),
```

Comment:
no need to add `cdb.`, see the require above (that returns the global instance), I think it's better to use the require approach than reaching for objects in the global namespace (which is more intended to give access to object from outside the lib).